samples/apps/copilot-chat-app/webapi/ServiceExtensions.cs

```diff
@@ -78,39 +85,50 @@ internal static IServiceCollection AddCors(this IServiceCollection services)
     }
 
     /// <summary>
-    /// Add authorization services
+    /// Add authentication services
     /// </summary>
-    internal static IServiceCollection AddAuthorization(this IServiceCollection services, IConfiguration configuration)
+    internal static IServiceCollection AddCopilotChatAuthentication(this IServiceCollection services, IConfiguration configuration)
```

Comment:
These methods were renamed to include "CopilotChat" in order to make it more obvious that they are customizations and not the .net Add* methods.